packages/e2e-tests/specs/editor/various/popovers.test.js

```diff
@@ -11,16 +11,16 @@ describe( 'popovers', () => {
 	describe( 'dropdown', () => {
 		it( 'toggles via click', async () => {
 			const isMoreMenuOpen = async () =>
-				!! ( await page.$( '.edit-post-more-menu__content' ) );
+				!! ( await page.$( '.interface-more-menu-dropdown__content' ) );
```

Comment:
Same again here. Could be a good opportunity to make these tests less dependent on implementation details such as class name.